src/lib/spinel/spinel.h

```diff
@@ -2897,6 +2898,32 @@ enum
      */
     SPINEL_PROP_THREAD_NEW_DATASET = SPINEL_PROP_THREAD_EXT__BEGIN + 40,
 
+    /// MAC CSL Period
+    /** Format: `S` - MTD build only
```

Comment:
These properties are for CSL Receiver. I think FTD can also work as a CSL Receiver.

Reply:
As stated in the spec (Thread 1.2.0 spec June 18, 2019).
Chapter 11.1.1: _An FTD may act as an Active Router, a REED, or a FED. An MTD may act as a MED, SED,
SSED or BED._
CSL Receiver is a feature for SSED, so MTD only.
Do you think that it makes sense for FTD which is never rx-off-when-idle ?

Reply:
FTD, MTD refers to device capability. Generally, MTD is has a subset function of FTD. Usually we don't use FTD to act as SED or SSED. But it doesn't mean it cannot act as a SED. For example, we have two FTDs. We can also make one of them act as SED and enjoy low power consumption.